app/controllers/repository_columns_controller.rb

```diff
@@ -36,85 +35,10 @@ def create_html
     end
   end
 
-  def create
-    @repository_column = RepositoryColumn.new(repository_column_params)
-    @repository_column.repository = @repository
-    @repository_column.created_by = current_user
-
-    respond_to do |format|
-      format.json do
-        if @repository_column.save
-          log_activity(:create_column_inventory)
-
-          if generate_repository_list_items(params[:list_items])
-            render json: {
-              id: @repository_column.id,
-              name: escape_input(@repository_column.name),
-              message: t('libraries.repository_columns.create.success_flash',
-                         name: @repository_column.name),
-              edit_url:
-                edit_repository_repository_column_path(@repository,
-                                                       @repository_column),
-              update_url:
-                repository_repository_column_path(@repository,
-                                                  @repository_column),
-              destroy_html_url:
-                repository_columns_destroy_html_path(@repository,
-                                                     @repository_column)
-            },
-            status: :ok
-          else
-            render json: {
-              message: {
-                repository_list_items:
-                  t('libraries.repository_columns.repository_list_items_limit',
-                    limit: Constants::REPOSITORY_LIST_ITEMS_PER_COLUMN)
-              }
-            }, status: :unprocessable_entity
-          end
-        else
-          render json: { message: @repository_column.errors.full_messages },
-                 status: :unprocessable_entity
-        end
-      end
-    end
-  end
-
-  def edit
+  def edit_html
```

Comment:
Why _html is needed here?

Reply:
To be consistent with names of other endpoints here (`index_html`, `create_html`, `destroy_html`), which are having the same role: displaying the content of `manage_column_modal`.

Reply:
it doesn't follow Rails naming